webflow-hacks/cms-hacks/3/3.ts

```diff
@@ -0,0 +1,14 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const JOBS_COUNTER_SELECTOR = '[fs-hacks-element="cms-jobs-counter"]';
+  const JOB_COUNT_SELECTOR = '[fs-hacks-element="hack3-job-count"]';
+
+  // get the amount (length) of elements with selector JOBS_COUNTER_SELECTOR
+  // and assign that amount to a constant named jobsCounter
+  const jobsCounter = document.querySelectorAll(JOBS_COUNTER_SELECTOR).length;
+
+  // select the element with data attribute JOB_COUNT_SELECTOR
+  const jobCount = document.querySelector<HTMLElement>(JOB_COUNT_SELECTOR);
+  if (!jobCount) return;
```

Comment:
I would swap the order of declaring `jobsCounter` and `jobCount`.

If `jobCount` does not exist on the page, the function returns. So there's no need to query the `jobsCounter` elements.